tools/buildsys/src/builder.rs

```diff
@@ -551,6 +664,7 @@ fn create_marker_dir(
         BuildType::Package => "packages",
         BuildType::Kit => "kits",
         BuildType::Variant => "variants",
+        BuildType::Repack => "repack",
```

Comment:
merge into the previous match arm:
```suggestion
        BuildType::Repack => "variant",
```

We want the repack markers to go into the same directory that the variant markers go into, so that cleanup works consistently for repacked images.